google_assistant/rootfs/etc/services.d/google-assistant/run

```diff
@@ -23,5 +24,5 @@ elif ! bashio::fs.file_exists "${CRED_JSON}"; then
 fi
 
 bashio::log.info "Starting Home Assistant GAssisant SDK..."
-exec python3 /usr/bin/hassio_gassistant.py \
-    "${CRED_JSON}" "${PROJECT_ID}" "${MODEL_ID}"
+python3 /usr/bin/hassio_gassistant.py \
```

Comment:
I've removed the `exec` command here to be constant with how the `hassio_oauth.py` is called (`python3 /usr/bin/hassio_oauth.py`).

It works great without it, but I'm not sure if this could have any impact on how the script is run with Docker. Let me know if I should revert it or not.
I'm not sure

Reply:
Please revert that. `exec` hands over the currently running process to that command (completely replacing it).